packages/hooks/package.json

```diff
@@ -33,5 +32,8 @@
     "@nft/api-graphql": "^1.0.0-beta.7",
     "@types/react": "^17.0.51",
```

Comment:
Should we also put the type of react as peerDep?
I'm not sure how the dev dependencies impact the deps of the app this lib will be installed in. Does npm do not install them? I have a feeling it may install them.

Reply:
No idea about this one. I believe this is not needed as I didn't see anyone using `@types/react` as a peer dependancy. Didn't face any issues without this one as well.